.vib/schema-registry/goss/goss.yaml

```diff
@@ -0,0 +1,8 @@
+file:
+  /opt/bitnami/schema-registry/etc/schema-registry/schema-registry.properties:
+    mode: "0664"
+    filetype: file
+    exists: true
+    contains:
+      - "listeners = http://0.0.0.0:8081"
```

Comment:
This kind of configuration (IP and port) should be in a `vars.yaml` file. Please create it.